test/functional/test-url-replacements.js

```diff
@@ -950,6 +950,12 @@ describes.sandboxed('UrlReplacements', {}, () => {
         '&a=b&d=PROM&e=PAGE_LOAD_TIME');
   });
 
+  it('should replace USER_AGENT', () => {
```

Comment:
userAgent should be replaced during [page load time](https://github.com/ampproject/amphtml/blob/master/test/functional/test-url-replacements.js#L466).  Please add the test around [line](https://github.com/ampproject/amphtml/blob/master/test/functional/test-url-replacements.js#L591)